packages/babel-core/cjs-proxy-dev.cjs

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable */
+module.exports = require("$repo-utils").USE_ESM
```

Comment:
Won't this file be preserved on release? We should not depend on `$repo-utils`.

Reply:
It's in `.npmignore`

Reply:
https://github.com/babel/babel/pull/15838/files#diff-82bd9573b4cb005e6b3183120e5379562c70d9c37e57893a8224f6a89dc77623R116
Will it still remain in `exports`?

Reply:
No, it's overwritten by https://github.com/nicolo-ribaudo/babel/blob/c85defd2174e0c80fba975e00577afc300e9e995/packages/babel-core/package.json#L97